plugins/producer_plugin/producer_plugin.cpp

```diff
@@ -499,8 +499,14 @@ class producer_plugin_impl : public std::enable_shared_from_this<producer_plugin
    block_timing_util::producer_watermarks            _producer_watermarks;
    pending_block_mode                                _pending_block_mode = pending_block_mode::speculating;
    unapplied_transaction_queue                       _unapplied_transactions;
+   alignas(hardware_destructive_interference_sz)
    std::atomic<int32_t>                              _max_transaction_time_ms; // modified by app thread, read by net_plugin thread pool
+   alignas(hardware_destructive_interference_sz)
    std::atomic<uint32_t>                             _received_block{0};       // modified by net_plugin thread pool
+   alignas(hardware_destructive_interference_sz)
+   std::atomic<fc::time_point>                       _last_other_vote_recevied;
+   std::atomic<fc::time_point>                       _last_producer_vote_received;
+   fc::microseconds                                  _production_pause_vote_timeout;
```

Comment:
maybe add a `_us` postfix.

Reply:
I generally go with `_us` and `_ms` for int types. `fc::microseconds` prevents misuse.